xml/Microsoft.Aspnet.Snapin/MMC_CONTROL_TYPE.xml

```diff
@@ -12,6 +12,9 @@
   <Base>
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
+  <ReturnValue>
```

Comment:
That section from msdn can be ignored. It's wrong there. Please remove this.

Reply:
@mairaw - section has been removed.